### PR TITLE
create cringe site

### PR DESCRIPTION
**Pull request by egatsak**
Created html, css and one script

### CODE REVIEW

**Review by egatsak — COMMENTED** [review]
added html css javascript
screenshot

**Comment by Vadimchesh** [comment]
@egatsak Could you please add a screenshot of your site to the description above?

**Comment by Vadimchesh** [comment]
Please delete all comments in the code

File: script.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+let button = document.querySelector('.button');
```
Comment (Vadimchesh) [inline script.js:1]: Why did you decide to use querySelector in this case? 
What would happen if you added a new button with the same class name to your site?

Comment (egatsak) [reply script.js:1]: I didn't explore so deeply - I just copied the code from the exercise at Yandex.Practicum

**Comment by egatsak** [comment]
> Please delete all comments in the code

Okay, but could you please explain what for?

**Comment by Vadimchesh** [comment]
> > Please delete all comments in the code
> 
> Okay, but could you please explain what for?

The main branch shouldn't contain any irrelevant comments. All comments should be for relevant cases.  
For example, you can add comments to explain why you decide to use this instead then that. You can provide some comments to explain your code further.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    </p><p class="p1">Второй плюс: теплый климат и морской воздух. Весна считается не лучшим сезоном, часто идут дожди, но даже 2-3 солнечных дней в неделю мне хватало, чтоб поправить уровень витамина Д. Рецепт хорошего настроения: утром кофе с видом на горы, а на закате – вино на пляже под шум волн.
+    </p><p class="p1">Третий пункт: пешая доступность ко всему, что только может понадобиться в городе, и близость природы: каждые выходные можно выезжать с друзьями в горы, на пикники в Ботанический сад, к водопадам и в небольшие походы.
+    </p><p class="p1">Сам по себе город очень маленький, логически делится на две части: атмосферный Старый Батуми с узкими мощеными улочками и невысокими зданиями и новая часть с высотками и апарт-отелями. Дома XIX века с итальянскими двориками в Старом городе проигрывают в комфорте бездушным небоскребам: иногда случаются перебои с электричеством или водой, а многие здания нуждаются в капитальном ремонте. В новой части Батуми огромный выбор студий со свеженьким ремонтом и видом на море, а старинные улочки можно оставить для ежедневных прогулок.</em></strong> <br>
```
Comment (Vadimchesh) [inline index.html:22]: One line of code should be placed on one screen.

Comment (Vadimchesh) [reply index.html:22]: Unexpected </em> tag

Comment (Vadimchesh) [reply index.html:22]: Unexpected closing tag "strong"

**Pull request merged**